src/components/byline.tsx

```diff
@@ -40,6 +40,7 @@ const anchorStyles = (kicker: string, inverted: string): SerializedStyles => css
 
 const commentStyles = (kicker: string): SerializedStyles => css`
     color: ${kicker};
+    width: 75%;
```

Comment:
Is this needed for the wider breakpoints?

Reply:
I think so yes, it could be more than 75% but I think that's a real edge case to have a byline that long